ibm_catalog.json

```diff
@@ -78,78 +117,87 @@
-            },
-            {
-              "key": "existing_monitoring_crn"
-            },
-            {
-              "key": "existing_kms_instance_crn",
-              "required": true
+              "key": "existing_scc_instance_crn"
             },
             {
-              "key": "existing_scc_instance_crn"
+              "key": "scc_instance_name"
             },
             {
-              "key": "existing_scc_cos_kms_key_crn"
+              "key": "scc_instance_tags",
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "type": "string"
+                }
+              }
             },
             {
-              "key": "kms_endpoint_type",
+              "key": "scc_service_plan",
               "options": [
                 {
-                  "displayname": "Public",
-                  "value": "public"
+                  "displayname": "Standard",
+                  "value": "security-compliance-center-standard-plan"
                 },
                 {
-                  "displayname": "Private",
-                  "value": "private"
+                  "displayname": "Trial",
+                  "value": "security-compliance-center-trial-plan"
                 }
               ]
             },
             {
-              "key": "scc_cos_key_ring_name"
+              "key": "existing_cos_instance_crn"
             },
             {
-              "key": "scc_cos_key_name"
+              "key": "cos_instance_name"
             },
             {
+              "key": "cos_instance_tags",
               "custom_config": {
-                "config_constraints": {
-                  "generationType": "2"
-                },
                 "grouping": "deployment",
                 "original_grouping": "deployment",
-                "type": "region"
-              },
-              "key": "cos_region",
-              "required": true,
-              "type": "string"
-            },
-            {
-              "key": "cos_instance_name"
+                "config_constraints": {
+                  "type": "string"
+                }
+              }
             },
             {
-              "key": "cos_instance_tags"
+              "key": "cos_instance_access_tags",
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "type": "string"
+                }
+              }
             },
             {
-              "key": "cos_instance_access_tags"
+              "key": "scc_cos_bucket_name"
             },
             {
-              "key": "scc_cos_bucket_name"
+              "key": "scc_cos_bucket_region",
+              "type": "string",
+              "custom_config": {
```

Comment:
I dont think we can use the general region picker for COS buckets. I think we need to create custom dropdown picker with only the supported regions for COS buckets. We need to make sure the dropdown has an option to pass `__NULL__` and this should be the default. The display name for null option can say something like "Same region as SCC instance"